### PR TITLE
Hid non-functional links on Engage project site

### LINKED ISSUE #5330: Comment out broken Site link on Engage project
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/7f0c132c96f71230b8935759e1f8711ccb340c0f/CONTRIBUTING.md).

### Overview
We need to comment out the non-functional "Site" link for the Engage project to prevent users from encountering dead links on our site.

### Details
The Engage project has links that are currently leading to non-existent pages. This includes the main "Site" link. This issue aims to temporarily hide this link from the website's visitors until we have the correct, functioning links in place.

### Action Items
- [x] Go to the [engage.md](https://github.com/hackforla/website/blob/gh-pages/_projects/engage.md) file in the GitHub repository.
- [x] Identify the lines containing **only** the "Site" link.
- [x] Comment out both the line beginning with ` - name: ...` as well as the line with `   url: ...` so the "Site" link will not be rendered on the live website. In Markdown, this can be done by placing a '#' at the start of each line. If you are using VSCode, you can highlight the line(s) to be commented out, then select 'Ctrl/Cmd' + '/' to automatically insert the '#'s.
- [x] Confirm that the "Site" link is no longer visible on the live site by running the site locally on Docker and provide before and after screenshots.

### Resources/Instructions
[Engage](https://www.hackforla.org/projects/engage) project page
Associated ER: https://github.com/hackforla/website/issues/4594

### PR DESCRIPTION
**Pull request by erinzz**
Fixes #5330 

### What changes did you make?
  - Hid non-functional links in Engage project card 

### Why did you make the changes (we will use this info to test)?
  - Currently the links lead to non-existent pages. We want to hide the links until we have functional pages built for each link. 

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

Screenshot 2023-08-29 at 5 39 56 PM

</details>

<details>
<summary>Visuals after changes are applied</summary>

Screenshot 2023-08-29 at 5 40 06 PM

</details>

### CODE REVIEW

**Comment by brianf4** [comment]
Review ETA: 6 PM 8/31/2023
Availability: 1-8 PM Thursday

File: _projects/engage.md [hunk 1]
```diff
@@ -30,19 +30,19 @@ leadership:
...
       github: 'https://github.com/eselkin'
     picture: https://avatars.githubusercontent.com/eselkin
-links:
-  - name: GitHub
-    url: 'https://github.com/hackla-engage'
-  - name: Slack
-    url: 'https://hackforla.slack.com/archives/C6JBH478W'
-  - name: Test Site
-    url: 'https://smstaging.engage.town'
-  - name: Site
```
Comment (brianf4) [inline _projects/engage.md]: Just line 40 to be commented out.

File: _projects/engage.md [hunk 2]
```diff
@@ -30,19 +30,19 @@ leadership:
...
-    url: 'https://sm.engage.town'
```
Comment (brianf4) [inline _projects/engage.md]: Just line 41 to be commented out.

**Review by brianf4 — CHANGES_REQUESTED** [review]
Hey @erinzz I left comments on the lines that should just be commented out rather than all the links!

**Comment by brianf4** [comment]
Hey @erinzz , I believe the issue wanted to just comment out the name for "Site" and the url for "Site". As those links are the dead ones. All the other links are functional. I left comments on the 'file changed' tab showing what lines should be commented out.

**Comment by mademarc** [comment]
Review ETA: 8/31/2023
Availability: 7:01PM

**Review by mademarc — DISMISSED** [review]
Hey @erinzz The changes are done well and the interface on the screenshots looks good as well.

**Comment by StephenTheDev1001** [comment]
@brianf4 @erinzz  From the issue "Comment out both the line beginning with  - name: ... as well as the line with    url: ...". I think he wants lines 38 - 41 commented out.

**Comment by erinzz** [comment]
Hey @brianf4, thanks for catching that! And, @StephenTheDev1001, thanks for clarifying which lines to comment out! Ty everyone (@mademarc too) for reviewing my pull request. 

I've corrected the changes and only hid the non-functional links while keeping the functional ones. See below for the new after visuals. 

Screenshot 2023-08-31 at 7 36 09 PM

**Review by StephenTheDev1001 — CHANGES_REQUESTED** [review]
Replace the image in the `Visuals after changes are applied` section with the new screenshot you captured and you're all set. Good job on your first pull request.

**Comment by brianf4** [comment]
> @brianf4 @erinzz From the issue "Comment out both the line beginning with - name: ... as well as the line with url: ...". I think he wants lines 38 - 41 commented out.

@StephenTheDev1001 

You are correct that lines 38 through 41 should be commented out but for this issue in particular it's just asking for lines 40 and 41 to be commented out. 

I only say this because I actually just worked and an [issue](https://github.com/hackforla/website/pull/5339) where they **only** wanted the "Test Site" `name` & `url` commented out.

**Review by brianf4 — CHANGES_REQUESTED** [review]
Please comment out **only** lines 40 & 41.

**Review by StephenTheDev1001 — CHANGES_REQUESTED** [review]
@brianf4 is right. Comment out lines 40 - 41 and update the screenshot on the PR.

**Pull request closed without merging**